README.md

```diff
@@ -31,14 +40,17 @@ The configuration passed to `Configure` can contain the following fields:
-| `aws.accessKeyId`       | AWS Access Key ID                  | yes      | "THE_ACCESS_KEY_ID" |
-| `aws.secretAccessKey`   | AWS Secret Access Key              | yes      | "SECRET_ACCESS_KEY" |
-| `aws.region`            | AWS SQS Region                     | yes      | "us-east-1"         |
-| `aws.queue`             | AWS SQS Queue Name                 | yes      | "QUEUE_NAME"        |
-| `aws.visibilityTimeout` | AWS SQS Message Visibility Timeout | yes      | "5"                 |
-| `aws.url`               | URL for AWS (internal use only)    | no       |                     |
+| name                              | description                                                                                                                                                              | required | example                                      |
+| --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | -------------------------------------------- |
+| `aws.accessKeyId`                 | AWS Access Key ID                                                                                                                                                        | yes      | "THE_ACCESS_KEY_ID"                          |
+| `aws.secretAccessKey`             | AWS Secret Access Key                                                                                                                                                    | yes      | "SECRET_ACCESS_KEY"                          |
+| `aws.region`                      | AWS SQS Region                                                                                                                                                           | yes      | "us-east-1"                                  |
+| `aws.queue` (as a sqs queue name) | AWS queue name                                                                                                                                                           | no       | "QUEUE_NAME"                                 |
+| `aws.queue` (as a go template)    | Go template that is evaluated at runtime against a record to obtain the queue name                                                                                       | no       | `{{ index .Metadata "opencdc.collection" }}` |
+| `aws.queue` (empty)               | Makes the connector fetch the queue name from the `opencdc.collection`                                                                                                   | no       |                                              |
+| `aws.queue.useQueue`              | Forces the connector to write to the specified queue name, ignoring the `opencdc.collection` metadata field. Only relevant when `aws.queue` is given as a sqs queue name | no       |
```

Comment:
By simplifying the usage and defaulting to `opencdc.collection` except when specified, we shouldn't have the need for this extra configuration parameter leaving the configuration much simpler.